pykickstart/commands/rhsm.py

```diff
@@ -96,9 +96,9 @@ def _getParser(self):
                         ``--proxy=[protocol://][username[:password]@]host[:port]``
                         """)
         op.add_argument("--server-hostname", metavar="<subscription_service_server_url>", version=RHEL8, required=False,
-                        help="Red Hat subscription service server URL.")
+                        help="Red Hat Satellite server URL (registration to Satellite is supported since RHEL 9.0).")
```

Comment:
The parameter says hostname, metavar says URL and the help too. So what is the expected value? `satellite.example.com` or `https://satellite.example.com/`?